src/Catty/Extension&Delegate&Protocol/Extensions/Project+Extension/Project+CustomExtensions.m

```diff
@@ -24,14 +24,15 @@
 #import "SpriteObject.h"
 #import "Brick.h"
 #import "Script.h"
+#import "Pocket_Code-Swift.h"
 
 @implementation Project (CustomExtensions)
 
 - (void)updateReferences
 {
-    NSArray <SpriteObject*> *allObjects = [[NSMutableArray alloc] initWithArray: self.allObjects];
+    NSArray <SpriteObject*> *allObjects = [[NSMutableArray alloc] initWithArray: self.scene.objects];
```

Comment:
Please replace with the `allObjects` method. It is important that later it contains all objects and not only the objects of one Scene.